dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, name, options=None):
         self._categories = defaultdict(int)
         self.__calculations = {}
         self._filter_properties_w_options(self.__calculations, options)
+        self._gini_impurity = 0
```

Comment:
should this be None initially, like other stats?

Reply:
Yes, good catch.